Assets/GameAssets/World/WorldScripts/Boat/WaterHealth.cs

```diff
@@ -0,0 +1,42 @@
+/*********************************************************************************************************************
+// File Name :         WaterHealth
+// Author :            Mark Hanson
+// Creation Date :     11/12/2024
+//
+// Brief Description : Manages the health of the water from harpoons
+*********************************************************************************************************************/
+
+using UnityEngine;
+
+/// <summary>
+/// Manages the health for the water from harpoons
+/// </summary>
+public class WaterHealth : BaseHealth
+{
+    /// <summary>
+    /// Overrides the base TakeDamage script to play a specific sfx is the
+    /// Water splashes from the harpoon.
+    /// </summary>
+    /// <param name="damage"> amount of incoming damage </param>
+    /// <param name="damageSource"> incoming damage source </param>
+    public override void TakeDamage(float damage, IBaseDamage damageSource)
```

Comment:
This seems like a really odd way of doing this. Why does the water require health? We have an interactable interface for when objects are hit by the harpoon. That seems like a better way of addressing that.

Reply:
I agree. If we do want to use this, then why bother making it take damage? We could just not use that variable, no?

Reply:
this way was addressed by andrea as a fast way to do it for now that we will change later